statistics/handle/lockstats/lock_stats.go

```diff
@@ -42,15 +42,20 @@ var (
 	useCurrentSession = []sqlexec.OptionFuncAlias{sqlexec.ExecOptionUseCurSession}
 )
 
+// TableInfo is the table info of which will be locked.
+type TableInfo struct {
+	PartitionInfo map[int64]string
+	// schema name + table name.
+	FullName string
+}
+
 // AddLockedTables add locked tables id to store.
 // - exec: sql executor.
-// - tidAndNames: table ids and names of which will be locked.
-// - pidAndNames: partition ids and names of which will be locked.
+// - tables: tables of which will be locked.
```

Comment:
```suggestion
// - tables: tables that will be locked.
```
I think the meanings are the same but simpler? :laughing:

Reply:
Updated.